Misc/NEWS.d/next/Core_and_Builtins/2025-05-08-11-22-57.gh-issue-132380._9vB7H.rst

```diff
@@ -0,0 +1,2 @@
+For free-threaded build, allow non-interned strings to be cached in the type
```

Comment:
I don't think this caches non-interned strings. It seems to me that it allows non-interned strings as the lookup key, but the cache still only contains interned strings.